### PR TITLE
linux4.14: update to 4.14.52.

### PR DESCRIPTION
**Pull request by pbui**

### CODE REVIEW

File: srcpkgs/linux4.14/template [hunk 1]
```diff
@@ -9,7 +9,7 @@ homepage="http://www.kernel.org"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/linux4.14/template]: GPL-2.0-only

**Comment by maxice8** [comment]
please add [ci skip] somewhere so it doesn't hog up travis for 50 minutes.

**Comment by pbui** [comment]
@maxice8 I've changed the license.  Does [ci skip] go in just the PR title or in the commit?

**Comment by maxice8** [comment]
On Fri, Jun 22, 2018 at 07:59:59AM -0700, Peter Bui wrote:
> @maxice8 I've changed the license.  Does [ci skip] go in just the PR title or in the commit?
> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/323#issuecomment-399472068

i put in the commit's comments.

**Comment by cr6git** [comment]
4.14.52 is out!

**Pull request merged**